### ISSUE TITLE
Stop requiring iOS 13

### ISSUE THREAD

**Issue by piptib** [TortugaPower/BookPlayer#630]
Best app for audiobooks with sensible privacy politics and without any unnecessary fuss
I can't use because Apple don't support my phone anymore.
Please consider not requiring iOS 13 for this app. It worked great before but now many users are unable to use it because some artificial restrictions. Does iOS 13 have something this important for this app existence? Is i truly necessary? If it is then why it wasn't not so long ago?
This inexplicable restriction makes me really sad because it forces me (and probably many others) to use other apps that aren't as good as this app. Not. Even. Close. :disappointed:

**Comment by GianniCarlo** [comment]
Hi @piptib! hmmmm I get what you mean, but iOS 13 brings many performance improvements for us (code-wise), and the oldest device support (iPhone 6s) was released almost 6 years ago now, and we figured that it would be 'safe' to require it. For more info on what we are leveraging from iOS 13+:
* Combine, the reactive framework from Apple, and it has allowed us to refactor the whole Player screen (#618) and from the optimization, rewinding and forwarding when the speed is different than 1x, it no longer staggers or blanks out. We started using Combine for the new Import screen as the first steps and testing out the framework before doing the first refactor. The library and playlist/folder screens will also be refactored to take advantage of Combine
* SF Symbols, not as important, but it allows us to delete some icons we were providing, and use the ones provided by the system
* Diffable data sources, this is will be yet another refactor regarding the tables and how they are updated, I have yet to test this, but it's one of the things that we can now do by requiring iOS 13

There are some other reactive frameworks that could be used so we could support iOS 12, but third party dependencies have their own API, and possible bugs as well, might as well go straight with Apple's solution 😕
